MCP-2612: Test partial BP readings from Lighthouse

jbregstone opened the pull request:
<!-- Ensure the PR title reflects the feature or bug name -->

## What was the problem?
<!-- brief description of how things worked before this PR -->
We were not testing partial readings were included on the evidence PDF in the end2end tests.

Associated tickets or Slack threads:
<!-- replace "000" with ticket number in both places -->
- [MCP-2612](https://amida.atlassian.net/browse/MCP-2612)

## How does this fix it?[^1]
<!-- description of how things will work after this PR -->

    Add some partial blood pressures to LH Mock Service.

    Add some more parsing to the PDF to find the partial blood pressures as expected.

## How to test this PR
- Step 1
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

au2 commented on app/src/end2endTest/java/gov/va/vro/end2end/VroV2Tests.java:
```diff
@@ -662,7 +663,13 @@ void testAutomatedClaimFullPositivePresumptive() {
    */
   @Test
   void testAutomatedClaimFullPositiveIncompleteBloodPressures() {
-    testAutomatedClaimFullPositive("380");
```
This is good but I hate to remove other checks in testAutomatedClaimFullPositive. Please take a look at what I did in https://github.com/department-of-veterans-affairs/abd-vro/pull/1362 and maybe return pdf from testAutomatedClaimFullPositive instead of startAutomatedClaim. Also I would another one for "-/50" from LH. And I would make sure there are no "-/-"s. Some version of Warren's fixes would have that issue earlier.

jbregstone commented:
> This is good but I hate to remove other checks in testAutomatedClaimFullPositive. Please take a look at what I did in #1362 and maybe return pdf from testAutomatedClaimFullPositive instead of startAutomatedClaim. Also I would add another one for "-/50" from LH. And I would make sure there are no "-/-"s. Some version of Warren's fixes had that issue earlier.

Do you want me to create a 'spec' like in the other PR you linked? I have added checks for the other BP's mentioned and changed testAutomatedClaimFullPositive to return the PDF.

au2 commented:
@jbregstone yes, that is what I wanted but that is already done in the develop. That probably why you have conflicts. Can you please clear the conflict.
